File: apps/dashboard/src/lib/tone.ts
import type { Tone } from "../types/iic";

export const TONE_LABELS: Record<Tone, string> = {
  terse: "Terse",
  conv: "Conversational",
  edu: "Educational",
};

export const TONE_DESCRIPTIONS: Record<Tone, string> = {
  terse: "Numbers first, no analogies. For trading hours.",
  conv: "Plain language with light explanations. Default.",
  edu: "Family-friendly. Analogies, no acronyms, company names instead of tickers.",
};

export const ALL_TONES: Tone[] = ["terse", "conv", "edu"];

File: apps/agent_secretary/secretary/tone.py
"""Tone modes (workflow 15 §2.2).

`terse` for the principal, `conversational` default, `educational` for family.
"""

from __future__ import annotations

from typing import Literal

Tone = Literal["terse", "conv", "edu"]

TONE_SUFFIXES: dict[Tone, str] = {
    "terse": (
        "Be terse. Numbers first. No analogies, no hedging, no greetings. " "Avoid adjectives."
    ),
    "conv": (
        "Plain language. Conversational, light explanations. "
        "Avoid jargon unless the reader uses it."
    ),
    "edu": (
        "Explain to a family member with no finance background. "
        "Use analogies. No acronyms. Spell out company names instead of tickers."
    ),
}


def suffix(tone: Tone) -> str:
    return TONE_SUFFIXES[tone]
